misc-utils/enosys.c

```diff
@@ -244,6 +239,12 @@ int main(int argc, char **argv)
 
 	INSTR(BPF_STMT(BPF_RET | BPF_K, SECCOMP_RET_ALLOW));
 
+	if (dump) {
+		if (write_all(STDOUT_FILENO, filter, (f - filter) * sizeof(filter[0])))
+			err(EXIT_FAILURE, _("Could not dump seccomp filter"));
+		return EXIT_SUCCESS;
```

Comment:
How portable is the filter? (I mean archs, kernel versions, etc.)

Reply:
Archs, not at all. The script will detect the mismatch and kill the process.

Every other dimension should be portable.